yt/frontends/enzo_e/fields.py

```diff
@@ -40,14 +55,76 @@ class EnzoEFieldInfo(FieldInfoContainer):
     def __init__(self, ds, field_list, slice_info=None):
         super().__init__(ds, field_list, slice_info=slice_info)
 
+        # setup nodal flag information
+        for field in NODAL_FLAGS:
+            if ("enzoe", field) in self:
+                finfo = self["enzoe", field]
+                finfo.nodal_flag = np.array(NODAL_FLAGS[field])
+
     def setup_fluid_fields(self):
-        super().setup_fluid_fields()
+        from yt.fields.magnetic_field import setup_magnetic_field_aliases
```

Comment:
Unless there's a specific reason (like circular imports ?) this import statement should live at the top of the module